tests/unit/script/htmlimageelement.rs

```diff
@@ -0,0 +1,103 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use script::test::DOMString;
+use script::test::sizes::{parse_a_sizes_attribute, Size};
+use style::media_queries::{MediaQuery, MediaQueryType};
+use style::media_queries::Expression;
+use style::servo::media_queries::{ExpressionKind, Range};
+use style::values::specified::{Length, NoCalcLength, AbsoluteLength, ViewportPercentageLength};
+
+
+pub fn test_length_for_no_default_provided(len: f32) -> Length {
+    let length = Length::NoCalc(NoCalcLength::ViewportPercentage(ViewportPercentageLength::Vw(len)));
+    return length;
+}
+
+#[test]
+fn no_default_provided() {
+    let mut a = vec![];
+    let length = test_length_for_no_default_provided(100f32);
+    let size = Size { query: None, length: length };
+    a.push(size);
+    assert_eq!(parse_a_sizes_attribute(DOMString::new(), None), a);
+    println!("{:?}", parse_a_sizes_attribute(DOMString::new(), None));
```

Comment:
Nit: Please remove `println!` usage.

Note for the future: `println!` is not useful in the testsuite because the stdout is captured by cargo. Instead, we should `panic!` so that cargo prints the captured stdout (which also has the panic message).